pkg/kwokctl/cmd/create/cluster/cluster.go

```diff
@@ -43,8 +43,13 @@ type flagpole struct {
 
 // NewCommand returns a new cobra.Command for cluster creation
 func NewCommand(ctx context.Context) *cobra.Command {
+	logger := log.FromContext(ctx)
 	flags := &flagpole{}
 	flags.KwokctlConfiguration = config.GetKwokctlConfiguration(ctx)
+	flags.Options.Log = internalversion.Log{
+		KlogLevel:     logger.ToKlogLevel(),
+		SeverityLevel: logger.ToLogSeverityLevel(),
+	}
```

Comment:
```suggestion
	flags.Options.Log = internalversion.Log{
		KlogLevel:     logger.ToKlogLevel(),
		SeverityLevel: logger.ToLogSeverityLevel(),
	}
```

Move to the installation phase